subsys/bootloader/Kconfig

```diff
@@ -117,6 +117,12 @@ config SB_DEBUG_SIGNATURE_PUBLIC_KEY_LAST
 	  first. This is meant to be used for testing looping through the
 	  public keys.
 
+config SB_DEBUG_NO_VERIFY_HASHES
```

Comment:
This is for debug testing that users of NCS will never need to perform, correct? Should we remove these from menuconfig visibility then without some backend Kconfig option that enables Nordic debug testing?

Reply:
That would be good, but I don't think we can. If we remove the prompt, they will not be modifyable via prj.conf.

Reply:
You can make a `tests/subsys/bootloader/bl_validation_ff_key/Kconfig` fil, which contains:
```
config SB_DEBUG_NO_VERIFY_HASHES
	bool
	default y

source "Kconfig.zephyr"
```

so please make this symbol promptless.

Reply:
@oyvindronningstad ^^

Reply:
Thanks @tejlmand, fixed.